components/usage/pkg/server/server.go

```diff
@@ -40,17 +40,7 @@ type Config struct {
 	DefaultSpendingLimit db.DefaultSpendingLimit `json:"defaultSpendingLimit"`
 
 	// StripePrices configure which Stripe Price IDs should be used
-	StripePrices StripePrices `json:"stripePrices"`
-}
-
-type PriceConfig struct {
-	EUR string `json:"eur"`
-	USD string `json:"usd"`
-}
-
-type StripePrices struct {
-	IndividualUsagePriceIDs PriceConfig `json:"individualUsagePriceIds"`
-	TeamUsagePriceIDs       PriceConfig `json:"teamUsagePriceIds"`
+	StripePrices apiv1.StripePrices `json:"stripePrices"`
```

Comment:
This doesn't belong into the `apiv1` package. Keep it here in the `server` package.

That's because the Price ID is just a configuraiton detail of the Service, rather than the Service defining this.

Reply:
Fair. Perhaps move it into the `stripe` package, that should break the cycle.